lib/samples/style_graphics_with_symbols/style_graphics_with_symbols.dart

```diff
@@ -0,0 +1,335 @@
+// Copyright 2024 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import 'package:arcgis_maps/arcgis_maps.dart';
+import 'package:flutter/material.dart';
+
+class StyleGraphicsWithSymbols extends StatefulWidget {
+  const StyleGraphicsWithSymbols({super.key});
+
+  @override
+  State<StyleGraphicsWithSymbols> createState() =>
+      _StyleGraphicsWithSymbolsState();
+}
+
+class _StyleGraphicsWithSymbolsState extends State<StyleGraphicsWithSymbols> {
```

Comment:
This class declaration needs to have `with SampleStateSupport`, and up with the "import" statements it needs
```dart
import '../../utils/sample_state_support.dart';
```

Reply:
Done @PaulAllanSturm, let me know if there's anything else.